pkg/etcd/etcd.go

```diff
@@ -323,6 +325,19 @@ func (e *ETCD) Register(ctx context.Context, config *config.Control, handler htt
 		return nil, err
 	}
 
+	tombstoneFile := filepath.Join(etcdDBDir(e.config), "tombstone")
+	if _, err := os.Stat(tombstoneFile); err == nil {
+		errCode, err := ioutil.ReadFile(tombstoneFile)
+		if err != nil {
+			return nil, err
+		}
+		if string(errCode) == "10" {
```

Comment:
What happens in the case where this might not equal 10?

Reply:
+1 to Brian's comment - can you explain the significance of this value? If it's an arbitrary constant, then give it a `const`. If other values are unexpected/unhandled, print a warning or something.

Reply:
we dont concern ourselves with other error codes, only this error code that requires an action

Reply:
I will add it as a constant, but there is no real reason behind the number itself, I am open to suggestions regarding error codes, what do you think it could be?

Reply:
If the content is immaterial, then can we just check for its existence like the other files that we just create as empty? Or is this value going to be important when checking the exit code when running in a pod for RKE2?

Reply:
well, since we cant really check for error code in the pod, because the pod is deleted from the cluster when the node is deleted, then we can check only for the existence of the file and only create it if the error is detected

Reply:
Can we see if the etcd folks are willing to accept the tombstone PR, or if they'd prefer to use an exit code to indicate the error? I would hate to have us cary forward customizations to etcd that upstream doesn't want to take.

Reply:
Agreed, I will open an issue with etcd and explain the issue presenting our changes there

Reply:
added the following issue to etcd upstream https://github.com/etcd-io/etcd/issues/12506 and waiting for reply